resources/[XyzzyRP]/lss-frakcje/policja.lua

```diff
@@ -2,18 +2,86 @@
 @author Lukasz Biegaj <wielebny@bestplay.pl>
 @author Karer <karer.programmer@gmail.com>
 @author WUBE <wube@lss-rp.pl>
+@author MrDadosz <mrdadosz@gmail.com>
 @copyright 2011-2013 Lukasz Biegaj <wielebny@bestplay.pl>
 @license Dual GPLv2/MIT
 @package MTA-XyzzyRP
 @link https://github.com/lpiob/MTA-XyzzyRP GitHub
 ]]--
 
+local function changeBinds(player, shouldWork)
+  if not isElement(player) then
+    return
+  end
+  if shouldWork then
+    toggleControl(player, "fire", true)
+    toggleControl(player, "aim_weapon", true)
+    toggleControl(player, "action", true)
+    toggleControl(player, "enter_exit", true)
+  else
+    toggleControl(player, "fire", false)
+    toggleControl(player, "aim_weapon", false)
+    toggleControl(player, "action", false)
+    toggleControl(player, "enter_exit", false)  
+  end
+end
+
 addEvent("onKajdankiZakuj", true)
 addEventHandler("onKajdankiZakuj", resourceRoot, function(kto,kogo)
-  triggerClientEvent(kogo, "onKajdankiZakuj", resourceRoot, kto)
+  local obecnie=getElementData(kogo, "kajdanki")
+  if not isElement(kto) or (obecnie and isElement(obecnie) and obecnie==kto) then
+  setElementData(kogo,"kajdanki", nil)
+  triggerClientEvent(kogo, "onKajdankiRozkuj", resourceRoot)
+  changeBinds(kogo, true)
+  if isElement(kto) then
+    outputChatBox(getPlayerName(kto) .. " zdejmuje z Ciebie kajdanki.")
+  end
+  return
+  end
+  setElementData(kogo, "kajdanki", kto)
+  triggerClientEvent(kogo, "onKajdankiZakuj", resourceRoot)
+  changeBinds(kogo, false)
+  outputChatBox(getPlayerName(kto) .. " zakuwa Cię w kajdanki.")
```

Comment:
Tutaj również powinno być:
```suggestion
    outputChatBox(getPlayerName(kto) .. " zakuwa Cię w kajdanki.", kogo)
```

aby wyeliminować błąd, przez który komunikat pokazuje się każdemu graczowi